### DISCUSSION TITLE
how to create table with merge cell via html

### DISCUSSION THREAD

**Discussion by whoisltd** [pymupdf/PyMuPDF#3988] [Looking for help]
I want to create table with merge cell via html. But can't find out how to do that.
`rowspan` or `colspan` not work on `td` tag 
Example table:
![image](https://github.com/user-attachments/assets/630a57da-1776-4fad-8020-fa90588eaeb2)

**Comment by JorjMcKie** [comment]
If you are sure that you made the correct HTML syntax for your table, then that is all that you can do.
If it is not handled correctly in `page.insert_htmlbox()` then there is nothing that can be done about it.

The `Story` object, that is invoked internally, fails to support this. There are indeed limits WRT `Story` support of HTML / CSS syntax. **_Roughly_** it is HTML4 and CSS2, but there are exceptions here and there. Among such examples is no support of text underlining.
